src/renderer/components/SettingsModal.tsx

```diff
@@ -206,6 +208,25 @@ const AppearanceSettings = memo(() => {
                     </NumberInputStepper>
                 </NumberInput>
             </SettingsItem>
+
+            <SettingsItem
+                description="The amount of padding for the viewport PNG export."
+                title="Viewport PNG export padding"
+            >
+                <NumberInput
+                    defaultValue={viewportExportPadding || 20}
+                    max={100}
+                    min={0}
+                    value={Number(viewportExportPadding || 0)}
```

Comment:
`viewportExportPadding` is a number already.
```suggestion
                    value={viewportExportPadding}
```